### PR TITLE
Hotfix for validation

### PR DESCRIPTION
**Pull request by anxolerd**
During I/O Extended Kyiv 2016 registration several issues were reported. Among them was validation failure with empty www field.

This pull request fixes empty www field was not validated and adds a testcase for that.

===
N.B. there is one more issue about `www` field validation. If website url have `+` sign in it, validation fails. 
Sample url: `https://plus.google.com/+OlexandrKovalchuk`

===

/cc @bluebirrrrd

### CODE REVIEW

File: src/GDGUkraine/lib/validation/field_validators.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 
 def email(field, value, error):
+    if not value:
+        return
```
Comment (bluebirrrrd) [inline src/GDGUkraine/lib/validation/field_validators.py]: why?

File: src/GDGUkraine/lib/validation/field_validators.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 
 def email(field, value, error):
+    if not value:
```
Comment (webknjaz) [inline src/GDGUkraine/lib/validation/field_validators.py]: why did you do this? why do you treat empty email as valid one?

**Comment by webknjaz** [comment]
@anxolerd ?

**Comment by anxolerd** [comment]
@webknjaz: code smells. 
Cannot write a good one right now. Thus closing PR w/o merging.

**Pull request closed without merging**